django_file_form/models.py

```diff
@@ -95,3 +96,18 @@ def __init__(self, file_id, **kwargs):
 
     def get_values(self):
         return dict(id=self.file_id, name=self.name)
+
+
+class PlaceholderUploadedFile(object):
```

Comment:
There is no easy way to add a flag to `UploadedFileWithId` since it requires a file object. It is therefore easier to use a new class with matching interface.